GetCertificate/handler.ts

```diff
@@ -44,6 +44,30 @@ const assertNever = (x: never): never => {
   throw new Error(`Unexpected object: ${toString(x)}`);
 };
 
+const replaceAll = (
+  str: string,
+  find: string | RegExp,
+  replace: string
+): string => str.replace(new RegExp(find, "g"), replace);
+
+export const toAnonymizedMessage = (
+  _errors: ReadonlyArray<ValidationError> | string,
+  fiscalCodeHash: string,
+  authCodeHash: string
+): string =>
```

Comment:
Code design:

this is a case when currying can be handy. As the last two parameters are more about configuration and they will be probably the same along the request, a signature like this may help:

```ts
toAnonymizedMessage = 
	(fiscalCodeHash: string, authCodeHash: string) =>
  		(_errors: ReadonlyArray<ValidationError> | string) => string
```

So you can 

```ts
const anonymize = toAnonymizedMessage(fiscal_code, authCodeSHA256)

ResponseErrorInternal(anonymize(e))

Reply:
Yes, I totally agree

Reply:
Done in [d3935fd](https://github.com/pagopa/io-functions-eucovidcerts/pull/41/commits/d3935fd11a339589db72d5f1abe26c3e0e33d41f)

I've also changed implementation to simplify it